app/controllers/api/v1/bulk_download_controller.rb

```diff
@@ -461,6 +462,12 @@ def self.load_study_files(ids: [], accessions: [], file_types: [])
           []
         end
       end
+
+      private
+
+      def bulk_download_params
+        params.require(:bulk_download).permit(file_ids: [], azul_files: {})
```

Comment:
since this is only used by the `auth_code` controller endpoint, should we rename this to `auth_code_params`?  Or just inline it inside that method?

Reply:
I went with `bulk_download` since that's the namespace the parameters are inside.  I also went with the private method since this is the standard implementation for strong parameters, and is consistent with all our other strong parameter instances.